__docs__/wonder-blocks-announcer/announcer.stories.tsx

```diff
@@ -36,29 +36,116 @@ const AnnouncerExample = ({
 type StoryComponentType = StoryObj<typeof AnnouncerExample>;
 
 /**
- * Announcer exposes an API for screen reader messages using ARIA Live Regions.
- * It can be used to notify Assistive Technology users without moving focus. Use
- * cases include combobox filtering, toast notifications, client-side routing,
- * and more.
+ * > 🌱 **Note:** This is a new package. We would love your feedback on it!
  *
- * Calling the `announceMessage` function automatically appends the appropriate live regions
- * to the document body. It sends messages at a default `polite` level, with the
- * ability to override to `assertive` by passing a `level` argument. You can also
- * pass a `debounceThreshold` to wait a specific duration before making another announcement.
+ * Announcer exposes an API for screen reader messages using [ARIA Live Regions](https://www.w3.org/TR/wai-aria/#attrs_liveregions).
+ * It can be used to notify Assistive Technology users without moving
+ * keyboard focus. Use cases include combobox filtering, toast notifications,
+ * client-side routing, and more.
  *
- * To test this API, turn on VoiceOver for Mac/iOS or NVDA on Windows and click the example button.
+ * Announcer is an ES6 class that creates a singleton instance. It can be
+ * optionally configured once per page load with the `initAnnouncer` function.
+ * Calling `announceMessage` will automatically create this instance if not
+ * already present.
+ *
+ * Messages alternate between one of two live regions per level (`polite` and `assertive`).
+ * They are automatically cleared from the DOM after a `5000` ms delay.
+ *
+ * To test the Announcer, turn on VoiceOver for Mac/iOS or NVDA on Windows and play with the [React example below](#react-announcer-example).
+ *
+ * By default, the `wbAnnouncer` container element is rendered _visually hidden_.
+ *
+ * > In Storybook, the Live Regions appended by Announcer are shown visually on the
+ * right side of the screen for debugging (see red outlined boxes). This can be
+ * controlled with a Storybook parameter, `addBodyClass: "showAnnouncer"`.
+ *
+ * ## API functions
+ * ### announceMessage()
```

Comment:
One suggestion would be to include the argument in the function signature so that it's clear it takes an options argument (since it's one object, rather than multiple parameters)!

```suggestion
 * ### announceMessage(options)
```

Reply:
Ooh, good call! And I forgot the required `message` option. Those are both in there now!